File: Sorting/Bubble Sort/bubble.py
#  function to perform the bubble sort

def bubblesort(arr):

    # outer loop for passes 
    for i in range(len(arr)):

        # inner loop for swapping elements
        # inside the array 
        for j in range(0, len(arr)-1-i):

            # swapping is performed if
            # a element is greater than next element
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]

a = [12, 90, 66, 34, 78, 51, 6]
bubblesort(a)

print("Bubble Sorted --> ", end=" ")
for i in range(len(a)):
    print(a[i], end=" ")

File: Sorting/Bubble Sort/bubble.java

public class bubble {
    
    // function for bubble sort
    void bubblesort(int arr[]) {
        
        for (int i = 0; i < arr.length-1; i++)
            for (int j = 0; j < arr.length-1-i; j++)
                if (arr[j] > arr[j+1]) {
                    
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
    }
    // prints the sorted array
    void printArray(int arr[]) {
        for (int i = 0; i < arr.length; i++)
            System.out.print(arr[i] + " ");
        System.out.println();
    }

    public static void main(String[] args) {
        bubble obj = new bubble();
        int arr[] = { 23, 90, 77, 4, 11, 70, 56 };
        obj.bubblesort(arr);
        System.out.println("Bubble Sorted --> ");
        obj.printArray(arr);
    }
    
}